error.go

```diff
@@ -21,20 +21,100 @@ type ErrorCode string
+	ErrorCodePaymentIntentIncompatiblePaymentMethod ErrorCode = "payment_intent_incompatible_payment_method"
+	ErrorCodePaymentIntentInvalidParameter          ErrorCode = "payment_intent_invalid_parameter"
+	ErrorCodePaymentIntentPaymentAttemptFailed      ErrorCode = "payment_intent_payment_attempt_failed"
+	ErrorCodePaymentIntentUnexpectedState           ErrorCode = "payment_intent_unexpected_state"
+	ErrorCodePaymentMethodUnactivated               ErrorCode = "payment_method_unactivated"
+	ErrorCodePaymentMethodUnexpectedState           ErrorCode = "payment_method_unexpected_state"
+	ErrorCodePayoutsNotAllowed                      ErrorCode = "payouts_not_allowed"
+	ErrorCodePlatformAPIKeyExpired                  ErrorCode = "platform_api_key_expired"
+	ErrorCodePostalCodeInvalid                      ErrorCode = "postal_code_invalid"
+	ErrorCodeProcessingError                        ErrorCode = "processing_error"
+	ErrorCodeProductInactive                        ErrorCode = "product_inactive"
+	ErrorCodeRateLimit                              ErrorCode = "rate_limit"
+	ErrorCodeResourceAlreadyExists                  ErrorCode = "resource_already_exists"
+	ErrorCodeResourceMissing                        ErrorCode = "resource_missing"
+	ErrorCodeRoutingNumberInvalid                   ErrorCode = "routing_number_invalid"
+	ErrorCodeSecretKeyRequired                      ErrorCode = "secret_key_required"
+	ErrorCodeSepaUnsupportedAccount                 ErrorCode = "sepa_unsupported_account"
+	ErrorCodeSetupAttemptFailed                     ErrorCode = "setup_attempt_failed"
+	ErrorCodeSetupIntentAuthenticationFailure       ErrorCode = "setup_intent_authentication_failure"
+	ErrorCodeSetupIntentUnexpectedState             ErrorCode = "setup_intent_unexpected_state"
+	ErrorCodeShippingCalculationFailed              ErrorCode = "shipping_calculation_failed"
+	ErrorCodeSkuInactive                            ErrorCode = "sku_inactive"
+	ErrorCodeStateUnsupported                       ErrorCode = "state_unsupported"
+	ErrorCodeTaxIDInvalid                           ErrorCode = "tax_id_invalid"
+	ErrorCodeTaxesCalculationFailed                 ErrorCode = "taxes_calculation_failed"
+	ErrorCodeTestmodeChargesOnly                    ErrorCode = "testmode_charges_only"
+	ErrorCodeTLSVersionUnsupported                  ErrorCode = "tls_version_unsupported"
+	ErrorCodeTokenAlreadyUsed                       ErrorCode = "token_already_used"
+	ErrorCodeTokenInUse                             ErrorCode = "token_in_use"
+	ErrorCodeTransfersNotAllowed                    ErrorCode = "transfers_not_allowed"
+	ErrorCodeUpstreamOrderCreationFailed            ErrorCode = "upstream_order_creation_failed"
+	ErrorCodeURLInvalid                             ErrorCode = "url_invalid"
+
+	// The following error codes can be returned though are undocumented
+	ErrorCodeIncorrectCVC     ErrorCode = "incorrect_cvc"
+	ErrorCodeInvalidCVC       ErrorCode = "invalid_cvc"
```

Comment:
I have no idea how this is even working, but both `ErrorCodeIncorrectCVC` and `ErrorCodeInvalidCVC` are already defined (with the same values) above.